README.md

```diff
@@ -47,17 +47,16 @@ If set to true you are accessing the headless Sauce instances (this discards the
 Type: `boolean`<br>
 Default: `false`
 
-## Usage
+### proxy
 
-All accessible API commands with descriptions can be found [here](docs/interface.md). In order to route requests through a proxy set `HTTP_PROXY`, `HTTPS_PROXY` or `NO_PROXY` as environment variable. For more information on this, read [here](https://github.com/request/request#controlling-proxy-behaviour-using-environment-variables). Alternatively you can pass the proxy to be used during initialization:
+If you want to tunnel your API request through a proxy please see the [got proxy docs](https://github.com/sindresorhus/got/blob/master/readme.md#proxies) for more information.
```

Comment:
Looking at got's README, you need to install either tunnel or global packages to be able to proxy the requests?

Reply:
I am not entirely sure what the common use case for users are therefor I thought the best would be to do a major release and have them figure out what works for them. I am happy to make it simpler moving on.